yuito-syz/goodbooking
# create-nuxt-app#217

## Nuxt初期設定
```sh
% npx create-nuxt-app new-frontend

create-nuxt-app v4.0.0
✨  Generating Nuxt.js project in new-frontend
? Project name: new-frontend
? Programming language: TypeScript
? Package manager: Npm
? UI framework: Vuetify.js
? Nuxt.js modules: Axios - Promise based HTTP client
? Linting tools: ESLint, Prettier, StyleLint
? Testing framework: Jest
? Rendering mode: Single Page App
? Deployment target: Static (Static/Jamstack hosting)
? Development tools: (Press <space> to select, <a> to toggle all, <i> to invert se
lection)
? Continuous integration: CircleCI
? What is your GitHub username? yuito-syz
? Version control system: Git
```

```sh
 Successfully created project new-frontend

  To get started:

        cd new-frontend
        npm run dev

  To build & start for production:

        cd new-frontend
        npm run build
        npm run start

  To test:

        cd new-frontend
        npm run test


  For TypeScript users. 

  See : https://typescript.nuxtjs.org/cookbook/components/
```
